lib/testOptions.ts

```diff
@@ -0,0 +1,60 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+
+/**
+ * Get the default mocha options for ads extension unit test. The following environment variables will be examined.
+ * ADS_TEST_GREP: custom test filter.
+ * ADS_TEST_INVERT_GREP: whether to invert the grep results.
+ * RUN_UNSTABLE_TESTS: whether to run the unstable tests. By default, only the stable tests will be executed unless this variable is set to true. If ADS_TEST_GREP is specified, this variable will be ignored.
```

Comment:
`whether to run unstable tests (tests that have @UNSTABLE@ in their name) ...`

Reply:
done